content/akeneo-event-platform/available-events.md

```diff
@@ -1,4 +1,989 @@
 # Available events
 
-::: panel-link <!-- TODO TEXT PANEL LINK -->[Next](/akeneo-event-platform/platform-usage-requirements.html)
-:::
\ No newline at end of file
+## com.akeneo.pim.v1.app.deleted
+
+*This event is generated whenever an App is deleted from the PIM.*
+
+* Message ID: `com.akeneo.pim.v1.app.deleted`
+* Content type: [application/json](https://www.iana.org/assignments/media-types/application/json)
+
+### Payload
+
+| Name | Type | Description |
+|---|---|---|
+| data.app | object | - |
+| data.app.client_id | string | Unique identifier of an App in the PIM. |
+
+> Example of payload
+
+
+```json
+
+{
+  "specversion": "1.0",
+  "id": "018e32f9-dfe4-760e-a273-5da1c089dfdb",
+  "type": "com.akeneo.pim.v1.app.deleted",
+  "source": "pim",
+  "subject": "srnt-whatever",
+  "time": "2024-02-22T17:31:00Z",
+  "datacontenttype": "application/json",
+  "dataschema": "https://event.sdk.akeneo.cloud/com.akeneo.pim.v1.app.deleted.schema.json",
+  "data": {
+    "app": {
+      "client_id": "test_client_id"
+    }
+  }
+}
+```
+
+## com.akeneo.pim.v1.attribute.deleted
+
+*This event is generated whenever an attribute is deleted from the PIM.*
+
+* Message ID: `com.akeneo.pim.v1.attribute.deleted`
+* Content type: [application/json](https://www.iana.org/assignments/media-types/application/json)
+
+### Payload
+
+| Name | Type | Description |
+|---|---|---|
+| data.attribute | object | - |
+| data.attribute.code | string | Unique identifier of an Attribute in the PIM. |
+
+> Example of payload
+
+
+```json
+
+{
+  "specversion": "1.0",
+  "id": "b6f4add4-f3d1-4502-8cf0-0f228e054ce5",
+  "type": "com.akeneo.pim.v1.attribute.deleted",
+  "source": "pim",
+  "subject": "srnt-whatever",
+  "time": "2024-06-10T17:31:00Z",
+  "datacontenttype": "application/json",
+  "dataschema": "https://event.sdk.akeneo.cloud/com.akeneo.pim.v1.attribute.deleted.schema.json",
+  "data": {
+    "attribute": {
+      "code": "an-attribute-code"
+    }
+  }
+}
+```
+
+## com.akeneo.pim.v1.attribute.updated
+
+*This event is generated whenever an existing attribute is modified in the PIM.*
+
+* Message ID: `com.akeneo.pim.v1.attribute.updated`
+* Content type: [application/json](https://www.iana.org/assignments/media-types/application/json)
+
+The modifications that can lead to this event include changes to the following properties:
```

Comment:
Personally i'm not fan of this listing :smiley: 

I would prefer:
* Change on any property of the attribute and save
* Attribute reordened in an attribute-group (all attributes in the group are considered updated)
* ...

Then it's easier to know what real action leads to an event

Reply:
do you know where I could find a complete version of this list?

Reply:
Unfortunately no real list.

For example a thread on the product https://akeneo.slack.com/archives/C032F5FAZ/p1723811026487079 that show how many trigger we have

Reply:
But it can be something we enrich as we discover them . 

Maybe we can ask also the team what they think

Reply:
i agree with mister @AdelELHAIBA-Akeneo first com :eyes: